test/e2e/cluster.go

```diff
@@ -119,9 +119,9 @@ func createStatefulSet(ctx context.Context, cli kubernetes.Interface) error {
 	oc, _ := clients.OpenshiftClusters.Get(ctx, vnetResourceGroup, clusterName)
 	installVersion, _ := version.ParseVersion(*oc.ClusterProfile.Version)
 
-	defaultStorageClass := "managed-premium"
-	if installVersion.V[0] == 4 && installVersion.V[1] == 11 {
-		defaultStorageClass = "managed-csi"
+	defaultStorageClass := "managed-csi"
+	if installVersion.Lt(version.NewVersion(4, 11)) {
+		defaultStorageClass = "managed-premium"
```

Comment:
Maybe this can be extracted to a function: _getDefaultStorageClass()_.

Reply:
This is a good idea. That said, I would rather do this later during improvement of storage class testing in general (eg.  listing all storage classes for given configuration along with info which one is the default). Moreover I have to admit I'm not sure where would I place this function.